src/ferry_planner/schedule.py

```diff
@@ -24,9 +25,11 @@ class FerrySailing(BaseModel):
     """Duration in seconds."""
     # TODO: price: float  # noqa: FIX002
     """Price in Canadian dollars (CAD)."""
+    notes: Sequence[str] | None
+    """Any notes/comments posted about this sailing"""
```

Comment:
```suggestion
    """Notes or comments posted about this sailing."""
```
Same applies to `FerrySchedule`